src/interfaces/configObserver.ts

```diff
@@ -1,3 +1,10 @@
+/**
+ * @interface ConfigObserver
+ * 
+ * This interface is used by the ConfingStorage inside the Solid backend. The
+ * purpose is to be notified when the Solid session information need to be
+ * stored.
+ */
 export interface ConfigObserver {
     onConfigChanged(config: string): void;
 }
```

Comment:
If this interface is only used for the Solid back-end and nothing else, then you can move it to the Solid module, where it doesn't even have to be exported.